build-tools/create-packs/Directory.Build.targets

```diff
@@ -70,8 +70,12 @@
     </ItemGroup>
   </Target>
 
+  <Target Name="BuildILLinkCustomStep">
+    <Exec Command="$(AndroidToolchainDirectory)\dotnet\dotnet build /p:Configuration=$(Configuration) $(MSBuildThisFileDirectory)..\..\src\Microsoft.Android.Sdk.ILLink\Microsoft.Android.Sdk.ILLink.csproj" />
```

Comment:
This could use `$(DotNetPreviewTool)`, like the others below.

I think you might also need `&quot;` around the path to the `.csproj`. If there were a space in the path it would break.